pallets/asset/src/lib.rs

```diff
@@ -437,12 +436,15 @@ decl_storage! {
                 ClassicTickers::insert(&import.ticker, classic_ticker);
             }
 
+            // Reserving country currency logic
+            let tconfig = || config.ticker_registration_config.clone();
```

Comment:
Reusing the classic ticker registration config doesn't seem right. Ostensibly we want the currency tickers to never expire?

Reply:
Hmm, I am not sure, @adamdossa can you lend some insight on this?

Reply:
Conclusion from yesterday's meeting, iirc: we should use a special `CurrencyReservation` issuer/DID, and the config here shouldn't allow expiry.